Makefile

```
@@ -218,16 +228,22 @@ binaries-linux_ppc64le:
 %-package:
 ifneq ($(SKIP_COMPILE), true)
 	$(MAKE) binaries-linux_$(ARCH)
+endif
+ifneq ($(filter $(ARCH), $(BUNDLE_SUPPORTED_ARCHS)),)
+ifneq ($(SKIP_BUNDLE), true)
+	$(MAKE) -C agent-bundle agent-bundle-linux ARCH=$(ARCH) DOCKER_REPO=$(DOCKER_REPO)
+endif
 endif
 	docker build -t otelcol-fpm internal/buildscripts/packaging/fpm
-	docker run --rm -v $(CURDIR):/repo -e PACKAGE=$* -e VERSION=$(VERSION) -e ARCH=$(ARCH) -e SMART_AGENT_RELEASE=$(SMART_AGENT_RELEASE) otelcol-fpm
+	docker run --rm -v $(CURDIR):/repo -e PACKAGE=$* -e VERSION=$(VERSION) -e ARCH=$(ARCH) otelcol-fpm
 
 .PHONY: msi
 msi:
 ifneq ($(SKIP_COMPILE), true)
 	$(MAKE) binaries-windows_amd64
 endif
-	./internal/buildscripts/packaging/msi/build.sh "$(VERSION)" "$(SMART_AGENT_RELEASE)" "$(DOCKER_REPO)"
+	test -f ./dist/agent-bundle_windows_amd64.zip || (echo "./dist/agent-bundle_windows_amd64.zip not found!" && exit 1)
```

Do you mind adding some guidance to the error for running ~`make.ps1 bundle`?

---

Thoughts?  Unfortunately, the windows bundle currently needs to be built natively on a windows host.
```suggestion
	test -f ./dist/agent-bundle_windows_amd64.zip || (echo "./dist/agent-bundle_windows_amd64.zip not found! Either download a pre-built bundle to ./dist/, or run './agent-bundle/scripts/windows/make.ps1 bundle' on a windows host and copy it to ./dist/." && exit 1)
```

---

That seems good to me, thanks.